sparse/mlir_backend/tests/test_simple.py

```diff
@@ -323,13 +300,9 @@ def test_coo_3d_format(dtype):
 
 @parametrize_dtypes
 def test_sparse_vector_format(dtype):
-    format = sparse.levels.get_storage_format(
-        levels=(sparse.levels.Level(sparse.levels.LevelFormat.Compressed),),
-        order="C",
-        pos_width=64,
-        crd_width=64,
-        dtype=sparse.asdtype(dtype),
-    )
+    if sparse.asdtype(dtype) in {sparse.complex64, sparse.complex128}:
+        pytest.xfail("Heisenbug")
```

Comment:
Let's give a one sentence description why we skip it (because in assertion first elements have incorrect values on python 3.10 IIRC). We can also `xfail` with `if` for the Python version where it fails.

Reply:
This is actually quite random on the Python version, but deterministic on previous code paths + data somehow. I don't fully know the reason. I'll add a description.